velox/exec/tests/QueryAssertionsTest.cpp

```diff
@@ -365,6 +365,49 @@ TEST_F(QueryAssertionsTest, nullDecimalValue) {
       "Types of expected and actual results do not match");
 }
 
+TEST_F(QueryAssertionsTest, valuesMismatch) {
+  auto test = makeRowVector({
+      makeFlatVector<int64_t>({10, 20}, DECIMAL(12, 2)),
+  });
+  createDuckDbTable({test});
+
+  // The expected values should be off by 1.
+  auto plan = PlanBuilder().values({test}).project({"c0"}).planNode();
+  EXPECT_NONFATAL_FAILURE(
+      assertQuery(plan, "SELECT c0 + 1 FROM tmp"),
+      "Expected 2, got 2\n2 extra rows, 2 missing rows\n2 of extra rows:\n\t0.10\n\t0.20\n\n2 of missing rows:\n\t1.10\n\t1.20");
```

Comment:
This error message is difficult to read. Would you split it into separate strings, one string per line?

Reply:
Maybe, put each line in a separate string and move "\n" to the end:

```
EXPECT_NONFATAL_FAILURE(
      assertQuery(plan, "SELECT c0 + 1 FROM tmp"),
      "Expected 2, got 2\n"
      "2 extra rows, 2 missing rows\n"
      ...);
```

Reply:
Fixed. Changed the same for all new tests implemented.

Reply:
@Krishna-Prasad-P-V this change seems to be missing. Can you check?

Reply:
Fixed.